src/modules/utils/motordrivercontrol/MotorDriverControl.h

```diff
@@ -60,6 +76,7 @@ class MotorDriverControl : public Module {
         uint32_t microsteps;
 
         char axis;
+        bool setup;
```

Comment:
remove this it is not needed and a waste of 4 bytes per instance

Reply:
ok, at the moment uart does not have restrictions using this driver